backupctl.spec

```diff
@@ -0,0 +1,47 @@
+%define __python /usr/bin/python3
+
+Name:           backupctl
+Version:        1.1.2
+Release:        1%{?dist}
+Summary:        Tool to manage zfs volumes and create new dirvish vault configurations.
+License:        GPLv3
+
+Source0:        https://github.com/adfinis-sygroup/%{name}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  python34 python34-setuptools python34-docutils
+BuildRoot:      %{_tmppath}/%{name}-%{version}
```

Comment:
BuildRoot is only needed for RHEL5 otherwise it gets set to a sane value per default.

You can check how macros are defined using rpm:

```bash
rpm -E '%buildroot'
```